src/main/content/_assets/js/builds.js

```diff
@@ -1106,19 +1106,22 @@ function displayMessage(message,javaMsg) {
         classNameIs = 'ind_starter_warning';
     }
     close_icon.on('click', function () {
-        $('.' + classNameIs).empty();
+        $('#starter_warnings').empty();
     });
     close_icon.on('keydown', function (event) {
         if ( event.which === 13 || event.which === 32 ) { // Enter key or spacebar
             $(this).click();
         }
     });
    
-    $('#starter_warnings').append("<div class='" + classNameIs +"'>");
+    $('#starter_warnings').append("<li class='" + classNameIs +"'>");
  
     $('.' + classNameIs)
-    .append(message)
-    .append(close_icon);  
+    .append(message);
+
+    $('#starter_warnings li:first-child').css("padding-top","20px");
```

Comment:
Can you move these styles to a css file since they don't seem to change based on logic